### ISSUE TITLE
Add months & years

### ISSUE THREAD

**Issue by sharkdp** [sharkdp/purescript-quantities#15]
Those are not perfectly well defined, but this seems reasonable:

```
1 year = 365 days

1 month = 30 days + 10 hours = 730 hours
-> 12 months = 365 days
```

**Comment by NOTtheMessiah** [comment]
This results in ambiguity by what is meant by a year. Do you mean typical Common (non-leap) year? Leap year? Julian year? Gregorian calendar year? Sidereal year? Tropical year?

It results in oddities such as a lightyear being inconsistently defined, as it uses a Julian year, whereas you use a common year.

**Comment by sharkdp** [comment]
> This results in ambiguity by what is meant by a year.

I don't think there is any ambiguity. A `year` in Insect/purescript-quantities is defined to be exactly equal to `365 days` (i.e. a "common year").

> It results in oddities such as a lightyear being inconsistently defined, as it uses a Julian year, whereas you use a common year.

Yes. The `lightyear` is defined to be *"the distance that light travels in vacuum in one Julian year (365.25 days)"*. I agree that it is unfortunate that

```
1 lightyear / year
```

is not equal to the speed of light (but slightly larger by a factor 365.25 / 365.0). But I don't think that this means that the lightyear is "inconsistently defined".

What do you propose? Change the definition of `year` to be equal to a Julian year?

In this case, I would probably also change the definition of a month to
```
1 month = 30 days + 10 hours + 30 minutes
```
such that we still have `12 months == 1 year`.

**Comment by sharkdp** [comment]
fixed in 92feca4ec665d129d40aa31dbd70da34414cb483
